.circleci/config.yml

```
@@ -42,6 +42,27 @@ jobs:
           command: |
             /scripts/integration-tests.sh
 
+  testPrometheusRules:
+    docker:
+      - image: web3f/ci-commons:v2.1.3
```

We can bump to the latest version
```suggestion
      - image: web3f/ci-commons:v2.4.0
```